vita3k/renderer/src/scene.cpp

```diff
@@ -223,44 +223,93 @@ COMMAND(handle_transfer_copy) {
     const SceGxmTransferType src_type = helper.pop<SceGxmTransferType>();
     const SceGxmTransferType dst_type = helper.pop<SceGxmTransferType>();
 
```

Comment:
The whole code below is valid if and only if src->format == dst->format (and the format is byte-aligned and not YUV)